public/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html ng-app="todoListApp">
+  <head>
+    <title>Things ToDo</title>
+    <script type="text/javascript" src="assets/angular/angular.min.js"></script>
+    <script type="text/javascript" src="assets/jquery/dist/jquery.min.js"></script>
+    <script type="text/javascript" src="js/app.js"></script>
+    <link href="style.css" rel="stylesheet">
+  </head>
+
+  <body>
+
+    <h1>Todo Today</h1>
+    <main>
+
+      <section ng-controller="TodoList as list">
+
+        <form name="addItem" ng-submit="addItem.$valid && list.addItem()" novalidate>
+
+          <input type="text" ng-model="list.newItem.content" required />
+          <br />
```

Comment:
Try not to use `<br>` tags - they are not relevant to the content of the page, but the visual aspect of the page, which should be controlled by CSS